src/services/dashboards/widgets/view-config.ts

```diff
@@ -0,0 +1,14 @@
+import { GROUP_BY } from '@/services/dashboards/widgets/config';
+
+export const GROUP_BY_ITEM_MAP = {
```

Comment:
Is there a reason you didn't put this code in widget/config.ts?
Is it because there is too much code in it?

---
이 코드를 widget/config.ts에 두지 않은 건 거기 너무 많은 코드가 있어서인가요?

Reply:
@yuda110 Yes.. It's toooooo big 🥲

Reply:
🥲